rust/rope/src/engine.rs

```diff
@@ -110,7 +119,7 @@ impl Engine {
 
     /// Get text of head revision.
     pub fn get_head(&self) -> Rope {
-        self.rev_content_for_index(self.revs.len() - 1)
+        self.text.clone()
```

Comment:
Sweet! Make me wonder though if the signature of this should be changed to `&Rope`